go/oasis-test-runner/test-runner.go

```diff
@@ -19,58 +18,7 @@ func main() {
 
 	// Register the e2e test cases.
 	rootCmd.Flags().AddFlagSet(e2e.Flags)
-	// Basic test.
-	_ = cmd.Register(e2e.Basic)
-	_ = cmd.Register(e2e.BasicEncryption)
-	// Byzantine executor node.
-	_ = cmd.Register(e2e.ByzantineExecutorHonest)
-	_ = cmd.Register(e2e.ByzantineExecutorWrong)
-	_ = cmd.Register(e2e.ByzantineExecutorStraggler)
-	// Byzantine merge node.
-	_ = cmd.Register(e2e.ByzantineMergeHonest)
-	_ = cmd.Register(e2e.ByzantineMergeWrong)
-	_ = cmd.Register(e2e.ByzantineMergeStraggler)
-	// Storage sync test.
-	_ = cmd.Register(e2e.StorageSync)
-	// Sentry test.
-	_ = cmd.Register(e2e.Sentry)
-	_ = cmd.Register(e2e.SentryEncryption)
-	// Keymanager restart test.
-	_ = cmd.Register(e2e.KeymanagerRestart)
-	// Dump/restore test.
-	_ = cmd.Register(e2e.DumpRestore)
-	// Halt test.
-	_ = cmd.Register(e2e.HaltRestore)
-	// Multiple runtimes test.
-	_ = cmd.Register(e2e.MultipleRuntimes)
-	// Registry CLI test.
-	_ = cmd.Register(e2e.RegistryCLI)
-	// Stake CLI test.
-	_ = cmd.Register(e2e.StakeCLI)
-	// Node shutdown test.
-	_ = cmd.Register(e2e.NodeShutdown)
-	// Gas fees tests.
-	_ = cmd.Register(e2e.GasFeesStaking)
-	_ = cmd.Register(e2e.GasFeesStakingDumpRestore)
-	_ = cmd.Register(e2e.GasFeesRuntimes)
-	// Identity CLI test.
-	_ = cmd.Register(e2e.IdentityCLI)
-	// Runtime prune test.
-	_ = cmd.Register(e2e.RuntimePrune)
-	// Runtime dynamic registration test.
-	_ = cmd.Register(e2e.RuntimeDynamic)
-	// Transaction source test.
-	_ = cmd.Register(e2e.TxSourceMultiShort)
-	_ = cmd.RegisterNondefault(e2e.TxSourceMulti)
-	// Node upgrade tests.
-	_ = cmd.Register(e2e.NodeUpgrade)
-	_ = cmd.Register(e2e.NodeUpgradeCancel)
-	// Debonding entries from genesis test.
-	_ = cmd.Register(e2e.Debond)
-
-	// Register the remote signer test cases.
-	rootCmd.Flags().AddFlagSet(remotesigner.Flags)
-	_ = cmd.Register(remotesigner.Basic)
+	_ = cmd.RegisterDefaultScenarios()
```

Comment:
This breaks framework/scenario separation, why do this?

Reply:
This was fixed by moving `cmp` to `oasis-test-runner` subcommand.